ncm-metaconfig/src/main/metaconfig/ganesha/pan/config_v1.pan

```diff
@@ -0,0 +1,17 @@
+unique template metaconfig/ganesha/config_v1;
+
+variable GANESHA_FSAL ?= undef;
+variable GANESHA_SERVICE ?= format('nfs-ganesha-%s',GANESHA_FSAL);
+variable CTDB_MANAGES_GANESHA ?= false;
+
+include 'metaconfig/ganesha/schema';
+
+bind "/software/components/metaconfig/services/{/etc/ganesha/ganesha.nfsd.conf}/contents" = ganesha_config;
+
+prefix "/software/components/metaconfig/services/{/etc/ganesha/ganesha.nfsd.conf}";
+"daemons" = {if (CTDB_MANAGES_GANESHA) { null } else { dict(GANESHA_SERVICE, "restart") }};
```

Comment:
@kwaegema when we apply this locally, we have to be careful to change this in our templates as well.